safe/definitions/hazard_exposure_specifications.py

```diff
@@ -55,7 +63,15 @@
                 'model.'),
             # end notes provided by Hadi Ghasemi
         ]
-    }
+    },
+    {
+        'hazard': hazard_flood,
+        'exposure': exposure_road,
+        'analysis_question': {
+            'fr': u"Dans l'événement d'une inondation, quelle longueur de "
```

Comment:
I didn't want you to translate this kind of strings on transifex. If you want, we can put the 'tr' if you think we need it. But as you told me these generated strings works well for you, I though it's fine.